pykern/pkcli/ci.py

```diff
@@ -35,23 +66,17 @@ def check_prints():
     wiki for an explanation of why this check is necessary.
 
     If you really need a print, use `pykern.pkconst.builtin_print`.
-
-    Args:
-        exclude (regex): path pattern to exclude from check
-    ..
```

Comment:
does `..` mean something? I saw you added it @robnagler f15946e86deddae7f27402ad41e73dc06cd19ba2

Reply:
No, that's a typo. I don't think there are any args to check_prints so please remove.